examples/3d-tiles/tile-3d-layer.js

```diff
@@ -135,14 +172,14 @@ export default class Tile3DLayer extends CompositeLayer {
     return changeFlags.somethingChanged;
   }
 
-  updateState({props, oldProps}) {
+  async updateState({props, oldProps}) {
```

Comment:
I think this is OK, but I am pretty sure that deck.gl does not expect its life cycle methods to be async...

Basically, another iteration of this layer could be calling `updateState` before this function returns.

Worth adding a short comment stating that we are "taking some liberties" here?